### ISSUE TITLE
Topics for cycle 2

### ISSUE THREAD

**Issue by murraycadzow** [smilefreak/studyGroup#24]
As mentioned yesterday we're looking for topics/presenters for the next 3 study group sessions

Topics that I had identified as good candidates and the default if there are no other suggestions
5 Nov - StudyGroup and Git/Github
19 Nov - Latex
3 Dec - R: subsetting and intro dplyr
(17 Dec - SYKSA?) - might be too busy or close to Christmas

So looking forward to comments and/or volunteers

**Comment by smilefreak** [comment]
I reckon flag the 17th of December one. It will reinvigorate the StudyGroup next year as the 5 minute lightning SYSKAS are exciting.

**Comment by murraycadzow** [comment]
Maybe 17 Dec becomes informal discussion/beers

**Comment by TomKellyGenetics** [comment]
I'm still recovering from dental surgery but I should be able to run (a basic refresher) on Git and Github next week. As discussed we'll use the example of adding yourself to the StudyGroup webpage.

Can we assume Software Carpentry (and having a GitHub profile) as a prerequisite or do we need to allow time to create profiles?

I'll be overseas in December so can't weigh in on then much. Maybe a less formal HackyHour meet up for anyone still in town?

**Comment by TomKellyGenetics** [comment]
I'd also stick to writing LaTeX stylised to be consistent and avoid confusion.

**Comment by murraycadzow** [comment]
Good point about stylisation, updated the comment to reflect so

**Comment by TomKellyGenetics** [comment]
This issue was moved to TomKellyGenetics/studyGroup#7

**Comment by smilefreak** [comment]
What happens here.

**Comment by TomKellyGenetics** [comment]
Just trying out moving issues between repos. The tool I've found (on the Google Github account) automatically posts and closes issues on the original repo. I'm not satisfied with it, I had the impression we could host it on any account but it's more difficult than it should be. Might be better to stick with everything here.

**Comment by murraycadzow** [comment]
I don't think moving and then closing is what you want to be doing, instead if you want the issues to be connected you should create a reference to the original issue. moving and closing is just going to create a weird tangle of where the thread actually lives

**Comment by TomKellyGenetics** [comment]
Yep, that sounds like what I wanted to do. Sorry, still new to GitHub. :)

I've got a handle on `fetch`/`pull`/`merge` now anyway but it's probably better to keep the central repo in one place so our new users can find it.

**Comment by TomKellyGenetics** [comment]
Anyway, sorry for the tangent - back on topic.

I understand Riku is back in December and available to do LaTeX (although I'll be away). I'd really like to see the LaTeX session so I'll admit I have a ulterior motive here. I'm happy for you guys to go ahead without me but I'm proposing a change in schedule for other reasons. Sorry for being a bit last minute.

As you know, we met up with Sung Bae yesterday and Peter Maxwell joined us for dinner (both from [NeSI](https://www.nesi.org.nz)). For those that don't know the New Zealand Infrastructure provides super-computing services to academics and companies across the country. Sung and Peter are support team members based in Christchurch and Dunedin respectively. They're keen to present a session on how to sign-up, access, and use servers or high performance computing. Peter has offered to do a SYSKA but only while he's based in Dunedin for the rest of this year. So we need to schedule a session with him soon if we're changing shedule.

We've also encouraged Peter to come to the Dunedin ResBaz next year (rather than the concurrent Auckland event) to get more NeSI presence here and present a similar session there too.

**Comment by rikutakei** [comment]
I'm so sorry, I should've mentioned earlier - my flight back is on 4th December. Is there any chance doing it at the start of next year?

**Comment by TomKellyGenetics** [comment]
It's ok Riku, I've already suggested an alternative to next session :) Mik or James are also able to sub-in for LaTeX if we're sticking with that.

I'm flying to Japan on December 1st, see ya there? :D
